frontend/src/forms/subforms/BuildingForm.tsx

```diff
@@ -74,20 +74,22 @@ const BuildingForm = <T extends any>(props: BuildingProps & FormikProps<T>) => {
     return lookupCode.type === API.OCCUPANT_TYPE_CODE_SET_NAME;
   }).map(mapLookupCode);
   const withNameSpace: Function = (name?: string) => {
-    return [props.nameSpace, `${props.index}`, name].filter(x => x).join('.');
+    return [props.nameSpace ?? '', `${props.index ?? ''}`, name].filter(x => x).join('.');
```

Comment:
Not required for this PR but would love to see this extracted to a utility helper. I believe we use this in several places.